examples/demo/src/dashboard/Dashboard.tsx

```diff
@@ -62,8 +63,7 @@ const Dashboard: FC = () => {
     );
 
     const fetchOrders = useCallback(async () => {
-        const aMonthAgo = new Date();
-        aMonthAgo.setDate(aMonthAgo.getDate() - 30);
```

Comment:
unfortunately, JS isn't smart enough to change month when we remove more days than the current day in the month.